docs/grammar/SolidityLexer.g4

```diff
@@ -184,7 +184,7 @@ fragment DoubleQuotedPrintable: [\u0020-\u0021\u0023-\u005B\u005D-\u007E];
   */
 fragment EscapeSequence:
 	'\\' (
-		['"\\bfnrtv\n\r]
+		['"\\nrt\n\r]
```

Comment:
@ekpyron Hmm, why is `n` twice ?

Reply:
That's
```
"\n"
```
vs
```
"\
"
```
isn't it?

Reply:
I have no idea. That was my explanation to myself, but you know the g4 lexer syntax much better.

Reply:
However we do not allow multiple string literals, or do we?

Reply:
Not sure if we *actually* allow ``\\\r``, though... or ``\\\r\n`` only, merging to one line break... or not even that - I didn't check. But ``\\\n`` is definitely allowed.

Reply:
Actually looking at ``Scanner::scanEscape`` it's probably right what the grammar does.

Reply:
> But \\\n is definitely allowed.

Isn't just `\` and newline?

Reply:
scanEscape calls tryScanEndOfLine which allows ``\n`` and ``\r``... so ``\`` plus newline as well as ``\`` plus carriage return seem to be valid in solidity.